deploy/handler/operator.yaml

```
@@ -229,6 +229,18 @@ webhooks:
         apiGroups: ["*"]
         apiVersions: ["v1alpha1","v1beta1"]
         resources: ["nodenetworkconfigurationpolicies", "nodenetworkconfigurationpolicies/status"]
+  - name: node-modified.nmstate.io
```

I suggest naming the hook node-labels-modified, so the it's clear what is the trigger (and also it would better fit the current naming convention)

```suggestion
  - name: node-labels-modified.nmstate.io
```